xline/src/main.rs

```diff
@@ -162,6 +156,19 @@ struct ServerArgs {
     jaeger_level: Option<LevelFilter>,
 }
 
+/// parse members from string
+fn parse_members(s: &str) -> Result<HashMap<String, String>, String> {
+    let mut map = HashMap::new();
+    for pair in s.split(',') {
+        if let Some((id, addr)) = pair.split_once('=') {
+            let _ignore = map.insert(id.to_owned(), addr.to_owned());
+        } else {
+            return Err("parse members error".to_owned());
+        }
+    }
+    Ok(map)
+}
+
```

Comment:
Can we deduplicate the code with that in benchmark?

Reply:
Add another utils crate? or let lock_utils become a more universal crate?

Reply:
I prefer to let `lock_utils` become more universal. The name of `lock_utils` seems to be too concrete,  or we can rename it `utils`. The `lock_utils` is one of the `utils`.